homeassistant/components/media_player/samsungtv.py

```diff
@@ -130,6 +133,10 @@ def get_remote(self):
 
     def send_key(self, key):
         """Send a key to the tv and handles exceptions."""
+        if self._is_powering_off and\
+                (key is 'KEY_POWER' or key is 'KEY_POWEROFF') is not True:
+            _LOGGER.info("TV is powering off, not sending command: %s", key)
+            return False
```

Comment:
It doesn't look like anything is checking the return value of this method.

Reply:
```
    def update(self):
        """Retrieve the latest data."""
        # Send an empty key to see if we are still connected
        return self.send_key('KEY')
```
The update method return a value, but I think it doesn't need to. I will remove the update return statement, and also remove the return statement of send_key().
Is that ok?

Reply:
Very good!